src/types/index.ts

```diff
@@ -16,6 +21,8 @@ export type ObjectValues<T> = T[keyof T];
 
 export type InputMessageStatus = ObjectValues<typeof INPUT_MESSAGE_STATUSES>;
 
+export type TxFunctionTypes = ObjectValues<typeof TX_FUNCTIONS>;
```

Comment:
This is not that simple. In some places we are using `snake_case` names and in other `camelCase`. I had a problem with this in one of my incoming PRs.

I solved this by using this types:
```
/**
 * TxFunction names coming directly from the API or ready to be sent.
 */
export type TxFunctionRaw = ObjectValues<typeof TX_FUNCTIONS>;

/**
 * TxFunctions used internally by the app.
 */
export type TxFunctionParsed = keyof typeof TX_FUNCTIONS;

export type TxFunction = TxFunctionRaw | TxFunctionParsed;
```

Here's the change I've made:
https://github.com/aeternity/superhero-wallet/pull/1780/files#diff-a9ba9cbedd19c9f66d564d2e89912890209b98f0a7ef19187877d2587300e476